config/taxon_list.csv

```diff
@@ -1,2 +1,2 @@
-cleaned_name,taxonomic_reference,cleaned_scientific_name_id,cleaned_name_taxonomic_status,cleaned_name_alternative_taxonomic_status,taxon_name,taxon_id,scientific_name_authorship,taxon_rank,taxonomic_status,family,taxon_distribution,establishment_means,scientific_name,scientific_name_id,taxonomic_resource
-Eucalyptus globulus,APC,https://id.biodiversity.org.au/name/apni/61221,accepted,NA,Eucalyptus globulus,https://id.biodiversity.org.au/taxon/apni/51440555,Labill.,Species,accepted,Myrtaceae,"WA (naturalised), SA, NSW, ACT (naturalised), Vic, Tas",native and naturalised,Eucalyptus globulus Labill.,https://id.biodiversity.org.au/name/apni/61221,NA
+original_name,aligned_name,taxon_name,taxon_name_alternatives,genus,family,taxon_rank,taxonomic_dataset,taxonomic_status,subclass,taxon_distribution,scientific_name,taxon_id,taxon_ID_genus,scientific_name_id,canonical_name,cleaned_scientific_name_id,aligned_name_taxonomic_status,aligned_name_taxon_id,taxon_id_family,trinomial,binomial,establishment_means
```

Comment:
From memory `taxon_ID_genus` is `taxon_id_genus` in austraits.build - based on the newer `build_update_taxon_list()` function